gensim/models/fasttext.py

```diff
@@ -41,7 +53,51 @@
     >>> fname = get_tmpfile("fasttext.model")
     >>>
     >>> model.save(fname)
-    >>> model = FastText.load(fname)  # you can continue training with the loaded model!
+    >>> model = FastText.load(fname)
+
+Once loaded, such models behave identically to those created from scratch.
+For example, you can continue training the loaded model:
+
+.. sourcecode:: pycon
+
+    >>> import numpy as np
+    >>> old_computer = np.copy(model.wv['computer'])  # Grab the existing vector for this word
+    >>> new_sentences = [
+    ...     ['computers', 'expensive'],
+    ...     ['computer', 'chess', 'players', 'stronger', 'than', 'humans'],
+    ...     ['computers', 'are', 'everywhere'],
+    ... ]
+    >>> model.train(new_sentences, total_examples=len(new_sentences), epochs=model.epochs)
+    >>> new_computer = model.wv['computer']
+    >>> # FIXME: why is this True??
```

Comment:
@mpenkov what's this about?

Reply:
@menshikh-iv and I are still trying to work out why the vectors still match after training. It's likely a broken a example.